cmake/Modules/Findrxcpp.cmake

```diff
@@ -13,13 +13,15 @@ set(URL https://github.com/Reactive-Extensions/rxcpp.git)
 set(VERSION a7d5856385f126e874db6010d9dbfd37290c61de)
 set_target_description(rxcpp "Library for reactive programming" ${URL} ${VERSION})
 
-
-if (NOT rxcpp_FOUND)
+# TODO 19.02.2019 lebdron: IR-346 Update Rx commit
+#if (NOT rxcpp_FOUND)
+  find_package(Git REQUIRED)
   externalproject_add(reactive_extensions_rxcpp
       GIT_REPOSITORY ${URL}
       GIT_TAG        ${VERSION}
       CONFIGURE_COMMAND ""
       BUILD_COMMAND ""
+      PATCH_COMMAND ${GIT_EXECUTABLE} apply ${PROJECT_SOURCE_DIR}/patch/0001-Fix-data-race-in-run-loop-usage.patch
       INSTALL_COMMAND "" # remove install step
       UPDATE_COMMAND "" # remove update step
```

Comment:
I am not sure, but seems that this line can prevent the existing cloned repo from updating to the version we need.